test/embedding/test-10-binding-basic.js

```diff
@@ -0,0 +1,32 @@
+const lwnode = process.lwnode;
+const port = process.lwnode.port;
+
+lwnode.ref();
```

Comment:
nit: This may not be necessary because `setInterval` will hold the loop.

Reply:
Okay. I removed it.